### PR TITLE
Feature for1780

### PR DESCRIPTION
**Pull request by HsbcJone**
[feature][plugin][sybase] datax support sybase plugins. #1780
 
 ## Check list
* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If any new Jar binary package adding in your PR, please add License Notice according
* [x] If necessary, please update the documentation to describe the new feature.
* [x] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. 
  2. Update the pom file of [datax](https://github.com/HsbcJone/DataX/blob/master/pom.xml)

### CODE REVIEW

**Comment by HsbcJone** [comment]
@dingxiaobo Hi, I have completed the plugin reading and writing for feature #1780 (datax that supports Sybase plugin). The code testing and documentation have been updated and verified. Please review.

feature is below:
https://github.com/alibaba/DataX/issues/1780

**Comment by HsbcJone** [comment]
@hf200012 @sangshuduo @iuhoay can you review

File: sybasereader/src/main/java/com/alibaba/datax/plugin/reader/sybasereader/SybaseReader.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import com.alibaba.datax.plugin.rdbms.reader.CommonRdbmsReader;
+import com.alibaba.datax.plugin.rdbms.reader.Constant;
+
+
+public class SybaseReader extends Reader {
+
```
Comment (penglin358) [inline sybasereader/src/main/java/com/alibaba/datax/plugin/reader/sybasereader/SybaseReader.java:17]: 麻烦补充一下Unit Test

Comment (HsbcJone) [reply sybasereader/src/main/java/com/alibaba/datax/plugin/reader/sybasereader/SybaseReader.java:17]: > 

hello 有模版吗？我看其他插件没有测试类。测试类需要增加哪些内容呢

Comment (HsbcJone) [reply sybasereader/src/main/java/com/alibaba/datax/plugin/reader/sybasereader/SybaseReader.java:17]: > > 
> 
> hello 有模版吗？我看其他插件没有测试类。测试类需要增加哪些内容呢

我已经增加unit test case.

**Review by sangshuduo — APPROVED** [review]
LGTM

**Comment by HsbcJone** [comment]
> @dingxiaobo Hi, I have completed the plugin reading and writing for feature #1780 (datax that supports Sybase plugin). The code testing and documentation have been updated and verified. Please review.
> 
> feature is below: #1780

@penglin358  还有其他补充吗？如果木有麻烦审核下咯

**Comment by HsbcJone** [comment]
> LGTM

thanks. Is there anyone else who can review it. 
@hf200012  @iuhoay @penglin358  @dingxiaobo

**Comment by HsbcJone** [comment]
> > LGTM
> 
> thanks. Is there anyone else who can review it. @hf200012 @iuhoay @penglin358 @dingxiaobo

@dingxiaobo this pr is done

**Comment by kevinwangcs** [comment]
Hi, May I kindly inquire about the reasons for removing the Neo4jWriter from pom.xml ? And there is no sybasewriter in pom.xml

**Comment by HsbcJone** [comment]
> Hi, May I kindly inquire about the reasons for removing the Neo4jWriter from pom.xml ? And there is no sybasewriter in pom.xml

fix done

**Review by kevinwangcs — APPROVED** [review]

**Pull request merged**